src/Compilers/Core/Portable/Operations/IConversionOperation.cs

```diff
@@ -40,7 +40,7 @@ public interface IConversionOperation : IOperation
         CommonConversion Conversion { get; }
         /// <summary>
         /// False if the conversion will fail with a <see cref="InvalidCastException"/> at runtime if the cast fails. This is true for C#'s
-        /// <code>as</code> operator and for VB's <code>TryCast</code> operator.
+        /// <c>as</c> operator and for VB's <c>TryCast</c> operator.
```

Comment:
Why didn't we use `<see langword="as" />` here?

Reply:
It's not part of our keywords that we provide in that completion list, which were taken [from here](https://github.com/EWSoftware/SHFB/blob/7e0304c76753d214f5350d9cc4afc720f77b6570/SHFB/Source/PresentationStyles/OpenXml/Transforms/MainSandcastle.xsl#L1128). It could be added since both languages offer equivalent semantics via a keyword either proactively (part of this issue) or as a separate change.